jigna/tests/test_jigna_web.py

```diff
@@ -1,23 +1,49 @@
 from threading import Thread
+import time
+import sys
 import unittest
 from unittest import skip
-import time
 
 try:
     from tornado.ioloop import IOLoop
-    from selenium import webdriver
 except ImportError:
     raise unittest.SkipTest("Tornado not installed")
 
+try:
+    from selenium import webdriver
+except ImportError:
+    raise unittest.SkipTest("selenium not installed")
+
 # Local imports.
-from jigna.api import Template, VueTemplate, WebApp
 from jigna.utils.web import get_free_port
 from test_jigna_qt import TestJignaQt, Person, body_html
 
 
+def patch_sys_modules():
```

Comment:
This won't do this as it does an exact match of 'PySide'/'PyQt' on the module keys.  I am checking for any substring so PySide.QtCore etc. will all be caught and removed.